Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -832,6 +834,118 @@ def list_device_groups_names():
     )
 
 
+def start_tsf_export():
+    """
+    Start export of tech support file (TSF) from PAN-OS:
+    https://docs.paloaltonetworks.com/pan-os/11-0/pan-os-panorama-api/pan-os-xml-api-request-types/export-files-api/export-technical-support-data
+    """
+    return_results('Starting TSF export...')
```

Comment:
```suggestion
```

this will exit the command execution and will not continue further

Reply:
It doesn't – I've tested it and it doesn't interrupt the flow. Let me know if I should still remove it.

Reply:
remove it, its not needed

Reply:
done.